Foundation/NSArray.swift

```diff
@@ -226,7 +226,7 @@ open class NSArray : NSObject, NSCopying, NSMutableCopying, NSSecureCoding, NSCo
         for idx in 0..<count {
             let item = _SwiftValue.store(self[idx])
```

Comment:
item here is a NSObject which is an object already; this should not bee need to be cast to `AnyObject?`